packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -208,6 +208,34 @@ class _LeastSquaresVelocityTrackerStrategy extends _VelocityTrackerStrategy {
 
 }
 
+/// A velocity in two dimensions.
+class Velocity {
```

Comment:
What's the advantage of that?  The point is to be explicit about the units, so the "x" accessor would then need to be something like `pixelsPerSecondX`...